Apps/WebClient/src/ClientApp/src/views/dependents.vue

```diff
@@ -106,9 +106,10 @@ export default class DependentsView extends Vue {
                             <b-col>
                                 <h5>
                                     You can add your dependents under the age of
-                                    12 to view their COVID-19 results. Minors
-                                    over the age of 12 can register for their
-                                    own Health Gateway accounts.
+                                    12 to view their COVID-19 results. Please
```

Comment:
age must be populated from the configuration

Reply:
Oh - i'm thinking we have to update this anyways, once we enable 12-18 access. 

How does one populate from config?